src/useCases/user/create/create-user.usecase.ts

```diff
@@ -1,48 +1,69 @@
 import { AppError, Report, StatusCode } from '@expressots/core'
```

Comment:
Se não está usando, remova!
```suggestion
import { StatusCode } from '@expressots/core'
```